tests/behat/features/gridfield-search.feature

```diff
@@ -23,3 +29,17 @@ Feature: Search in GridField
       And I click "Walmart" in the "#Form_EditForm" element
     Then I should see "Walmart"
       And I should see "Walmart" in the ".breadcrumbs-wrapper" element
+
+  Scenario: I can see pagination after search
+    When I go to "/admin/gridfield-test-navigation"
+      And I should see "Walmart" in the "#Form_EditForm" element
+      And I press the "Open search and filter" button
+      And I press the "Advanced" button
+      And I fill in "Search__Category" with "Oil"
+      And I press the "Enter" key in the "Search__Category" field
+    Then I should see "ExxonMobil" in the "#Form_EditForm" element
+      But I should not see "Walmart" in the ".col-Name" element
+      And I should not see "Vitol" in the ".col-Name" element
+      And I should see "1" in the ".pagination-page-number" element
```

Comment:
```suggestion
      And I should see an "#action_pagination_prev[disabled]" element
      And I should see an "#action_pagination_next" element
      And I should not see an "#action_pagination_next[disabled]" element
```
`And I should see "1" in the ".pagination-page-number" element` failed on my local because `1` is the attribute value rather than a text value. Seems like there isn't an existing way to test for attribute values within elements, though the way I've recommended test essentially the same thing

Reply:
Yeah, sure because it should test total number of pages. Replaced to `"2"`.
Updated.